CodeGeneration/Sources/SyntaxSupport/DeclNodes.swift

```diff
@@ -2006,11 +2006,11 @@ public let DECL_NODES: [Node] = [
     traits: ["WithStatements"],
     children: [
       Child(
-        name: "Statements",
+        name: "statements",
         kind: .collection(kind: .codeBlockItemList, collectionElementName: "Statement")
       ),
       Child(
-        name: "EndOfFileToken",
+        name: "endOfFileToken",
         deprecatedName: "EOFToken",
```

Comment:
Of course! Here's a revised version of your message for clarity and grammar:

I'm uncertain about the optimal way to handle this edge case.

If we want to make this `deprecatedName` lowercase as well, I'm not sure there's a better solution than simply hardcoding the condition `if deprecatedName == "eofToken" { ... ` somewhere 😕

Reply:
I wasn't suggesting this approach because I was thinking that we don't want to break the capability layer. However, if you're saying that we can do such changes, then I'm on board.

Here is the exact diff in the generated files:
```diff
diff --git a/Sources/SwiftSyntax/generated/RenamedChildrenCompatibility.swift b/Sources/SwiftSyntax/generated/RenamedChildrenCompatibility.swift
index 2df170c1..972b1377 100644
--- a/Sources/SwiftSyntax/generated/RenamedChildrenCompatibility.swift
+++ b/Sources/SwiftSyntax/generated/RenamedChildrenCompatibility.swift
@@ -7027,7 +7027,7 @@ extension SomeOrAnyTypeSyntax {

 extension SourceFileSyntax {
   @available(*, deprecated, renamed: "unexpectedBetweenStatementsAndEndOfFileToken")
-  public var unexpectedBetweenStatementsAndEOFToken: UnexpectedNodesSyntax? {
+  public var unexpectedBetweenStatementsAndEofToken: UnexpectedNodesSyntax? {
     get {
       return unexpectedBetweenStatementsAndEndOfFileToken
     }
@@ -7047,7 +7047,7 @@ extension SourceFileSyntax {
   }

   @available(*, deprecated, renamed: "unexpectedAfterEndOfFileToken")
-  public var unexpectedAfterEOFToken: UnexpectedNodesSyntax? {
+  public var unexpectedAfterEofToken: UnexpectedNodesSyntax? {
     get {
       return unexpectedAfterEndOfFileToken
     }
@@ -7062,9 +7062,9 @@ extension SourceFileSyntax {
       leadingTrivia: Trivia? = nil,
       _ unexpectedBeforeStatements: UnexpectedNodesSyntax? = nil,
       statements: CodeBlockItemListSyntax,
-      _ unexpectedBetweenStatementsAndEOFToken: UnexpectedNodesSyntax? = nil,
+      _ unexpectedBetweenStatementsAndEofToken: UnexpectedNodesSyntax? = nil,
       eofToken: TokenSyntax = .endOfFileToken(),
-      _ unexpectedAfterEOFToken: UnexpectedNodesSyntax? = nil,
+      _ unexpectedAfterEofToken: UnexpectedNodesSyntax? = nil,
       trailingTrivia: Trivia? = nil

   ) {
@@ -7072,9 +7072,9 @@ extension SourceFileSyntax {
         leadingTrivia: leadingTrivia,
         leadingTrivia: leadingTrivia,
         unexpectedBeforeStatements,
         statements: statements,
-        unexpectedBetweenStatementsAndEOFToken,
+        unexpectedBetweenStatementsAndEofToken,
         endOfFileToken: eofToken,
-        unexpectedAfterEOFToken,
+        unexpectedAfterEofToken,
         trailingTrivia: trailingTrivia
       )
   }
diff --git a/Sources/SwiftSyntaxBuilder/generated/RenamedChildrenBuilderCompatibility.swift b/Sources/SwiftSyntaxBuilder/generated/RenamedChildrenBuil
derCompatibility.swift
index ecaee9bc..53132ddb 100644
--- a/Sources/SwiftSyntaxBuilder/generated/RenamedChildrenBuilderCompatibility.swift
+++ b/Sources/SwiftSyntaxBuilder/generated/RenamedChildrenBuilderCompatibility.swift
@@ -751,9 +751,9 @@ extension SourceFileSyntax {
   public init(
       leadingTrivia: Trivia? = nil,
       unexpectedBeforeStatements: UnexpectedNodesSyntax? = nil,
-      unexpectedBetweenStatementsAndEOFToken: UnexpectedNodesSyntax? = nil,
+      unexpectedBetweenStatementsAndEofToken: UnexpectedNodesSyntax? = nil,
       eofToken: TokenSyntax = .endOfFileToken(),
-      unexpectedAfterEOFToken: UnexpectedNodesSyntax? = nil,
+      unexpectedAfterEofToken: UnexpectedNodesSyntax? = nil,
       @CodeBlockItemListBuilder statementsBuilder: () throws -> CodeBlockItemListSyntax,
       trailingTrivia: Trivia? = nil
     ) rethrows {
@@ -761,9 +761,9 @@ extension SourceFileSyntax {
         leadingTrivia: leadingTrivia,
         unexpectedBeforeStatements,
         statements: statementsBuilder(),
-        unexpectedBetweenStatementsAndEOFToken,
+        unexpectedBetweenStatementsAndEofToken,
         endOfFileToken: eofToken,
-        unexpectedAfterEOFToken,
+        unexpectedAfterEofToken,
         trailingTrivia: trailingTrivia
       )
   }
```
Is `EofToken` acceptable?

Reply:
Yes, that’s acceptable.